rdt/transformers/categorical.py

```diff
@@ -264,7 +264,10 @@ def _fit(self, data):
         else:
             freq = data.value_counts(normalize=True, dropna=False)
 
+        nan_value = freq[np.nan] if np.nan in freq.index else None
```

Comment:
I'm not sure this works with other types of nans, like float('nan')

Reply:
Maybe but this should not happen because freq is defined by `data.value_counts()`, no?

Reply:
I was concerned with the freq.index having nans in there which are not np.nan. I'm not sure if it's impossible for that to happen.